examples/btc-to-evm-fungible-transfer/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Please apply formatting

Reply:
I will wait for the tsconfig PR #487 to rebase this

Reply:
use configuration similar to `tsconfig.base.json`